code/controllers/UserController.php

```diff
@@ -66,27 +67,35 @@ public function sendInvite($data, Form $form)
         }
 
         $invite = UserInvitation::create();
-        $form->saveInto($invite);
-        try {
-            $invite->write();
-        } catch (ValidationException $e) {
+        if ($invite->canCreate(Member::CurrentUser()) == true){
```

Comment:
`canCreate` is the interface that Silverstripe typically uses for checking if a user can create a object, for things like ModelAdmin it is checked. Basically this is adding that check step. The validation step probably isn't the right spot as it isn't user input that has caused the problem.